userguide.md

```diff
@@ -132,7 +139,7 @@ You may explicitly include the path of pg_config to `PATH`, such as
 ```bash
 cd plr
 PATH=/usr/pgsql-9.4/bin/:$PATH; USE_PGXS=1 make
-echo ’PATH=/usr/pgsql-9.4/bin/:$PATH; USE_PGXS=1 make install’ | sudo sh
+echo "PATH=/usr/pgsql-9.4/bin/:$PATH; USE_PGXS=1 make install" | sudo sh
```

Comment:
Why double quotes? and I guess if we are updating we might as well use a recent version of postgresql ie psql-13

Reply:
* double quotes allows for shell variable substitution
* yes, the path to pg 13 can go in there instead